src/tuwien/auto/calimero/dptxlator/DptXlator16BitSet.java

```diff
@@ -75,6 +75,10 @@ public class DptXlator16BitSet extends DPTXlator {
 	}
 
 	// HVAC domain
+	public enum DhwControllerStatus implements EnumBase<DhwControllerStatus> {
+		Fault, Load, LegioProtection, Push, OtherEnergySource, SolarEnergyOnly, SolarEnergySupport, TemperatureOptimizationShift;
+	}
+	public static final EnumDpt<DhwControllerStatus> DptDhwControllerStatus = new EnumDpt<>("22.100", DhwControllerStatus.class);
```

Comment:
In the README, there are currently the two existing DPTs listed (22.101 and 1000). Can you please update the README (either with the new DPTs, or better yet, like the other lines with an example like "e.g., RHCC status, DHW controller status"